### PR TITLE
Javascript/javascript2/week3

### PR DESCRIPTION
**Pull request by adnan832**

### CODE REVIEW

**Comment by tpetrina** [comment]
@adnan832 This PR has no files in them.

**Comment by adnan832** [comment]
> @adnan832 This PR has no files in them.

click on the commits "added js2/week3 homework" you will find the files

**Comment by dpfernandes** [comment]
@adnan832 it doesn't work like that. A review has to be done on the files not on the commits. Please add an empty file and commit the files again so the files become visible on the PR.

**Comment by adnan832** [comment]
> @adnan832 it doesn't work like that. A review has to be done on the files not on the commits. Please add an empty file and commit the files again so the files become visible on the PR.

its fixed now :-)

File: javascript/javascript2/week3/fastPresserGameFun/main.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    else if(lCounter === sCounter){
+        sResult.textContent = 'draw, you can start again!';
+        lResult.textContent = 'draw, you can start again!';
+    }
+}, inputTime * 1000);
+});
+})
+reBtn.addEventListener('click', () => {
+    window.location.reload();
+});
```
Comment (zuhairtaha) [inline javascript/javascript2/week3/fastPresserGameFun/main.js:58]: I think you should re-read the question of this exercise and try to make it more correct

**Review by zuhairtaha — COMMENTED** [review]
God job. I added some comments. Feel free to update then I'll check it again.

**Pull request merged**